### ISSUE TITLE
Missing 'alt' attribute in the image tag

### ISSUE THREAD

**Issue by rb0946** [manishmanda29/a11y-ui#8]
Missing 'alt' attribute in the image tag: In HTML, it's a best practice to provide an 'alt' attribute for images. This attribute is essential for accessibility and SEO purposes.
